packages/backend/migration/1681429921400-Event.js

```diff
@@ -0,0 +1,95 @@
+export class Event1681429921400 {
+    name = 'Event1681429921400'
+
+    async up(queryRunner) {
+        await queryRunner.query(`DROP INDEX "public"."IDX_2cd3b2a6b4cf0b910b260afe08"`);
```

Comment:
The migration generated appears to have a bunch of extraneous modifications. Should these be removed so that only the new Event related modifications are included?